src/mumble/Audio.cpp

```diff
@@ -32,7 +35,7 @@ void CodecInit::initialize() {
 		oCodec->report();
 		g.oCodec = oCodec;
 	} else {
-		qWarning("CodecInit: Failed to load Opus, it will not be available for encoding/decoding audio.");
+		Log::logOrDefer(Log::CriticalError, QLatin1String("CodecInit: Failed to load Opus, it will not be available for encoding/decoding audio."));
```

Comment:
Instead of `QLatin1String` I would use `tr`, so that the string is translatable.